src/main.cpp

```diff
@@ -157,14 +158,17 @@ int main(int argc, char** argv) {
                "\t -D <file dump folder>     Folder where extracted files should be stored at\n"
                "\t -o <format>               Output format (default or csv)\n"
                "\t -x <plugin>               Don't activate the specified plugin\n"
+#ifdef ENABLE_PLUGIN_PROCTRACER
+               "\t -P <proctracer config>    Proctracer config json location\n"
+#endif
 #ifdef DRAKVUF_DEBUG
                "\t -v                        Turn on verbose (debug) output\n"
 #endif
         );
         return rc;
     }
 
-    while ((c = getopt (argc, argv, "r:d:i:e:t:D:o:vx:")) != -1)
+    while ((c = getopt (argc, argv, "r:d:i:e:t:D:o:vx:P")) != -1)
```

Comment:
This this should be `P:` as it's more then just a flag, it has a value that is supposed to be passed as option argument.